DigitalLearningSolutions.Web/Views/TrackingSystem/Delegates/ViewDelegate/ConfirmRemoveFromCourse.cshtml

```diff
@@ -0,0 +1,43 @@
+﻿@inject IConfiguration Configuration
+@using DigitalLearningSolutions.Web.Models.Enums
+@using DigitalLearningSolutions.Web.ViewModels.TrackingSystem.Delegates.ViewDelegate
+@using Microsoft.Extensions.Configuration
+@model RemoveFromCourseViewModel
+
+@{
+  var errorHasOccurred = !ViewData.ModelState.IsValid;
+  ViewData["Title"] = $"Remove delegate enrolment from {Model.CourseName}";
+  var cancelLinkRouteData = new Dictionary<string, string> { { "delegateId", Model.DelegateId.ToString() } };
+}
+
+<div class="nhsuk-grid-row">
+  <div class="nhsuk-grid-column-full">
+    @if (errorHasOccurred)
+    {
+      <vc:error-summary order-of-property-names="@(new[] { nameof(Model.Confirm) })" />
+    }
+    <h1 class="nhsuk-heading-xl nhsuk-u-margin-bottom-3">Remove delegate enrolment from @Model.CourseName</h1>
+    <h2 class="nhsuk-heading-l">Delegate: @Model.Name</h2>
+    <div class="nhsuk-form-group">
+      <form asp-action="ExecuteRemoveFromCourse" asp-route-delegateId="@Model.DelegateId" asp-route-customisationId="@Model.CustomisationId">
+        <fieldset class="nhsuk-fieldset" aria-describedby="example-hint">
+          <div class="nhsuk-checkboxes">
+            <div class="nhsuk-checkboxes__item">
+              <input type="hidden" name="name" value="@Model.Name" />
+              <input type="hidden" name="courseName" value="@Model.CourseName" />
+              <input class="nhsuk-checkboxes__input" id="confirm" name="confirm" type="checkbox" value="true" aria-describedby="confirm-item-hint">
+              <label class="nhsuk-label nhsuk-checkboxes__label" for="confirm">
+                I am sure that I want to remove @Model.Name from the @Model.CourseName course.
+              </label>
+              <div class="nhsuk-hint nhsuk-checkboxes__hint" id="confirm-item-hint">
+                This action will remove the course from the delegate's Current/Completed courses list and archive their progress.

```

Comment:
Will this actually remove the course from the delegate's Completed course list? We don't edit any completed progress records so I don't think it will. This might need to be confirmed and/or reworded.

Reply:
Good catch. I'll change this wording and we should ask them about this when we demo it.